crates/bevy_window/src/window.rs

```diff
@@ -594,6 +594,10 @@ pub struct WindowDescriptor {
     pub resize_constraints: WindowResizeConstraints,
     pub scale_factor_override: Option<f64>,
     pub title: String,
+    #[cfg(any(feature = "x11", feature = "wayland"))]
+    pub desktop_id: String,
+    #[cfg(feature = "x11")]
+    pub desktop_instance: String,
```

Comment:
```suggestion
	#[cfg(all(
            any(
                target_os = "linux",
                target_os = "freebsd",
                target_os = "netbsd",
                target_os = "openbsd"
            ),
            any(feature = "x11", feature = "wayland")
        ))]
    pub desktop_id: String,
    #[cfg(all(
            any(
                target_os = "linux",
                target_os = "freebsd",
                target_os = "netbsd",
                target_os = "openbsd"
            ),
            feature = "x11"
        ))]
    pub desktop_instance: String,
```

Reply:
Okay, I changed platform configuration, now committing.

Reply:
Why is it restricted to linux and {free,net,open}bsd? Is it not supported on macOS with X for example? Or on other systems using X natively?